lib/xebow/rgb_matrix.ex

```diff
@@ -15,22 +15,29 @@ defmodule Xebow.RGBMatrix do
   @sof <<0, 0, 0, 0>>
   @eof <<255, 255, 255, 255>>
 
-  @rows 4
-  @cols 3
-
-  # approximates ~60 FPS
-  @delay_ms 17
+  # pixels on the xebow start in upper left corner and count down instead of
+  # across
+  @pixels [
+    {0, 0},
+    {0, 1},
+    {0, 2},
+    {0, 3},
+    {1, 0},
+    {1, 1},
+    {1, 2},
+    {1, 3},
+    {2, 0},
+    {2, 1},
+    {2, 2},
+    {2, 3}
+  ]
```

Comment:
This code is probably low risk since there's only 1 variant of the Keybow (the 12 button model), but it strikes me as more fragile than the solution it's replacing. For example, someone could accidentally add two `{0, 0}` tuples to this list and it would cause a bug. It seems easier to unintentionally cause a bug by manually editing this list than say, setting the total pixel count.

Since this code should have a low likelihood of being modified, the example below is just a recommendation if you feel like going down this path.

```ex
@pixel_count 12
@columns 3

def get_yx(pixel_id) do
  x = rem(pixel_id, @columns)
  y = div(pixel_id, @columns)

  {y, x}
end
```

And if you add a couple helper functions...

```ex
def pixel_count_to_range(pixel_count) do
  (0..pixel_count-1)
end

def pixel_count_to_yx(pixel_count) do
  pixel_count
  |> pixel_count_to_range()
  |> Enum.map(&get_xy/1)
end
```

The animation code could look something like this:

```ex
@pixel_count
|> pixel_count_to_yx()
|> Enum.map(fn {y, x} ->
  # Animation code ...
end)
```

Another benefit to this approach is that it may be easier to abstract out the animation library if you ever wanted to support a range of 1D, 2D, and 3D LED matrix devices. It could be possible to build out generic animations and use them across many different hardware platforms.